### ISSUE TITLE
I cannot watch any games in dota 2

### ISSUE THREAD

**Issue by Danok59** [ValveSoftware/Dota2-Gameplay#3909]
### Description

I try to watch live TI games and other pro player games but I couldn't watch all of the live games in dota2. I had to wait like 2 min and then it redirects back to the home screen page. That infinity loop is happening whenever I try to watch dota2 games. Can you please help me what is happening to me. If I need to provide you something to investigate further please let me know.
Thank you in advance.

**Comment by TheUnholyWrath** [comment]
go read https://github.com/ValveSoftware/Dota2-Gameplay/issues/3692#issuecomment-1272324261

**Comment by TheUnholyWrath** [comment]
yeah its a bug but you're welcome to close this as a duplicate. #3639 is the original issue
